index.html

```diff
@@ -445,6 +452,14 @@ <h2>NEL policies</h2>
       </p>
       <p>
 
+      <p>
+      Each <a>NEL policy</a> has a list of <dfn data-lt-noDefault
+      data-lt="policy request headers">request headers</dfn> and a list of <dfn
+      data-lt-noDefault data-lt="policy response headers">response
+      headers</dfn>, each of which is a list of
+      header names.

```

Comment:
To be consistent with IETF nomenclature and be more precise, let's use "header field names" throughout?

Reply:
I was following Fetch, which uses "[header name](https://fetch.spec.whatwg.org/#concept-header-name)".  (Though I should link this to the fetch definition)